src/charts/CytoViz/components/ElementData/ElementData.js

```diff
@@ -14,9 +14,15 @@ const _generateAttributeDetails = (classes, labels, attributeName, attributeValu
   if (typeof attributeValue === 'object' && 0 in attributeValue) {
     // List represented as an object
     return (
-      <div key={attributeName} className={classes.attributeColumnContainer}>
-        <div className={classes.attributeLabel}>{attributeLabel}:</div>
-        <div className={classes.tableContainer}>
+      <div
+        data-cy={`cytoviz-element-attribute-${attributeName.toLowerCase()}`}
+        key={attributeName}
+        className={classes.attributeColumnContainer}
+      >
+        <div data-cy={`cytoviz-element-attribute-name`} className={classes.attributeLabel}>
+          {attributeLabel}:
+        </div>
+        <div data-cy={`cytoviz-element-attribute-value`} className={classes.tableContainer}>
```

Comment:
For two selectors above, can we use ordinary quotation marks instead of backticks ? :slightly_smiling_face:

Reply:
Nice catch :+1: